source/host/System.hh

```diff
@@ -1,30 +1,35 @@
 #pragma once
 
 #include <egg/core/SceneManager.hh>
-#include <test/TestDirector.hh>
 
-#include <span>
+#if defined(__arm64__) || defined(__aarch64__)
+inline void FlushDenormalsToZero() {
+    uint64_t fpcr;
+    asm("mrs %0,   fpcr" : "=r"(fpcr));
+    asm("msr fpcr, %0" ::"r"(fpcr | (1 << 24)));
+}
+#elif defined(__x86_64__) || defined(_M_X64) || defined(__i386__) || defined(_M_IX86)
+#include <immintrin.h>
+
+inline void FlushDenormalsToZero() {
+    _MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_ON);
+}
+#endif
 
 namespace Host {
 
 /// @addr{0x802A4080}
 class KSystem {
-private:
-    enum class Option {
-        Invalid = -1,
-        Suite,
-    };
-
 public:
-    int main(int argc, char **argv);
-    [[nodiscard]] Option option(char *arg);
-    void handleOption(Option opt, int argc, char **argv, int &i);
+    void main(int argc, char **argv);
     void init();
     void initMemory();
-    [[nodiscard]] bool run();
+    void run();
+    void calc();
 
     [[nodiscard]] EGG::Heap *rootHeap() const;
-    [[nodiscard]] const Test::TestDirector *testDirector() const;
+    [[nodiscard]] const EGG::SceneManager *sceneManager() const { return m_sceneMgr; };
+    [[nodiscard]] EGG::SceneManager *sceneManager() { return m_sceneMgr; };
```

Comment:
@vabold We don't have inline getters in this header, is this fine or should bodies be moved to source file?

Reply:
If these are exposed to external users it is a little faster for them to be inline

Reply:
At that rate we might want to inline a bunch of other getter functions that users will end up being exposed to. Such as in `KartState.hh`. Anyway let's wait on what vabold says to do